src/main/java/uk/gov/hmcts/ccd/domain/service/common/CaseEventEnablingService.java

```diff
@@ -24,4 +33,24 @@ public Boolean isEventEnabled(String enablingCondition,
         }
         return Boolean.TRUE;
     }
+
+    public Boolean isEventEnabled(String enablingCondition,
```

Comment:
I'm not sure a separate method is an appropriate way for this, bearing in mind that the general "ShowCondition" feature is pretty flexible. If we had even a slightly more complex scenario such as `SomeCaseDataField = "A" AND [INJECTED_DATA.otherField] = "B"`, then looks like this isn't going to have access to the case data too to be able to evaluate `SomeCaseDataField`?

It might be easier (and work for any kind of chaining) to modify the existing method in this class to take an argument such as just the injection metdata/an initial map containing just the injection metadata, and merge with `caseDetails.getCaseDataAndMetadata()`?

Reply:
Fixed in https://github.com/hmcts/ccd-data-store-api/pull/1985/commits/e9775ac2efe226ec855e75b4e136f32f16967b80